### DISCUSSION TITLE
zh/guide/drivers/163music

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#368] [Announcements]
# zh/guide/drivers/163music

网易云音乐云盘需要在手机端才可以进行上传音乐等操作，网页端无法操作 Cookie 进入 https://music.163.com 然后打开F12开发者模式 随意打开一个请求只要携带 Cookie 找到以下两个参数填写进AList的 Cookie 中就可以使用 __csrf 、MUSIC_U; 歌曲获取上限 默认只能获取200首歌曲，如果你超出可以修改...

https://alist.nn.ci/zh/guide/drivers/163music.html

**Comment by iowong** [comment]
歌词链接是用的http，如果alist是https的歌词就加载不了
这个错误信息“Mixed Content”表明你的网页在通过HTTPS安全协议加载时，尝试通过不安全的HTTP协议请求了一些资源。在现代的Web安全标准中，这种做法是不被允许的，因为HTTP连接可能容易受到中间人攻击，从而危及用户数据的安全。
在你的具体例子中，你的网页（https://.flac）是通过HTTPS加载的，但它尝试通过XMLHttpRequest请求一个HTTP资源（http://.lrc）。浏览器为了用户安全，自动阻止了这种混合内容的请求。

**Comment by TotoQWQ** [comment]
管理端没有网易云选项啊
